### ISSUE TITLE
Static UI content not installing

### ISSUE THREAD

**Issue by kjlynn** [DefectDojo/django-DefectDojo#482]
### Issue Type 
- [X] Bug
- [ ] Enhancement

### Description
I'm seeing the exact same UI issue as described in closed request #453: static UI elements don't appear to be installed.  --Sorry, I wasn't able to figure out how to reopen the closed issue. 

I was able to reproduce the issue several times as follows:

1. do a fresh install of Ubuntu desktop 16.04.2 on a VMware VM
2. run through the instructions in DefectDojo-Installation-Guide-Ubuntu-Desktop-16.04-2 
3. the result is that DoJo is up and running on port 8000, but all graphic UI elements are missing (see screenshot in case #453)

I tried running    **python manage.py collectstatic --clear**
but it has no effect on the issue.

After the install, a yarn-error.log is in the components folder.  The log indicates that the git fails:
**Arguments: ls-remote --tags --heads git://github.com/adobe-webplatform/eve.git
Directory: /home/klynn/django-DefectDojo/components
Output: fatal: unable to connect to github.com**
I tried adding this before the install:
**git config --global url."https://".insteadOf git://**
but no luck, same error in yarn_error.log

Any help appreciated

**Comment by devGregA** [comment]
@kjlynn I'm not able to reproduce this issue. Can you include your console log please?

**Comment by kjlynn** [comment]
Sure. Here's a transcript of the installations steps.  These were the first things done on a fresh install of Ubuntu 16.04.3 LTS amd64

[DefectDojo-Install.txt](https://github.com/DefectDojo/django-DefectDojo/files/1775333/DefectDojo-Install.txt)

**Comment by maennel** [comment]
@kjlynn Are you in a corporate environment that blocks outgoing SSH connections? 
I noticed some components installed with yarn have dependencies retrieved via git://...
I can reproduce this.

**Comment by maennel** [comment]
Here's a more detailed trace:
```
$ yarn install --verbose
[...]
verbose 44.731 Request "https://registry.yarnpkg.com/jade" finished with status code 200.
warning @yarn_components/mocha > jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
verbose 44.76 Request "https://registry.yarnpkg.com/commander" finished with status code 200.
verbose 44.768 Request "https://registry.yarnpkg.com/glob" finished with status code 200.
verbose 44.774 Performing "GET" request to "https://registry.yarnpkg.com/minimatch".
verbose 44.776 Performing "GET" request to "https://registry.yarnpkg.com/graceful-fs".
verbose 44.778 Performing "GET" request to "https://registry.yarnpkg.com/inherits".
verbose 44.785 Request "https://registry.yarnpkg.com/growl" finished with status code 200.
verbose 44.789 Request "https://registry.yarnpkg.com/mkdirp" finished with status code 200.
verbose 44.798 Request "https://registry.yarnpkg.com/diff" finished with status code 200.
verbose 44.803 Request "https://registry.yarnpkg.com/ms" finished with status code 200.
verbose 44.875 Request "https://registry.yarnpkg.com/inherits" finished with status code 200.
verbose 44.879 Request "https://registry.yarnpkg.com/graceful-fs" finished with status code 200.
warning @yarn_components/mocha > glob > graceful-fs@2.0.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
verbose 44.887 Request "https://registry.yarnpkg.com/minimatch" finished with status code 200.
warning @yarn_components/mocha > glob > minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
verbose 44.891 Performing "GET" request to "https://registry.yarnpkg.com/lru-cache".
verbose 44.893 Performing "GET" request to "https://registry.yarnpkg.com/sigmund".
verbose 44.976 Request "https://registry.yarnpkg.com/sigmund" finished with status code 200.
verbose 45.087 Request "https://registry.yarnpkg.com/lru-cache" finished with status code 200.
verbose 45.09 Performing "HEAD" request to "https://github.com/moment/moment".
verbose 45.929 Request "https://github.com/moment/moment" finished with status code 200.
verbose 45.931 Performing "GET" request to "https://github.com/moment/moment.git/info/refs?service=git-upload-pack".
verbose 46.767 Request "https://github.com/moment/moment.git/info/refs?service=git-upload-pack" finished with status code 200.
verbose 46.781 Performing "GET" request to "https://raw.githubusercontent.com/moment/moment/7c4c091bedb86ca6c1a5c5419ad5204ef3926347/package.json".
verbose 47.009 Request "https://raw.githubusercontent.com/moment/moment/7c4c091bedb86ca6c1a5c5419ad5204ef3926347/package.json" finished with status code 200.
verbose 47.019 Performing "HEAD" request to "https://github.com/morrisjs/morris.js".
verbose 47.798 Request "https://github.com/morrisjs/morris.js" finished with status code 200.
verbose 47.799 Performing "GET" request to "https://github.com/morrisjs/morris.js.git/info/refs?service=git-upload-pack".
verbose 48.368 Request "https://github.com/morrisjs/morris.js.git/info/refs?service=git-upload-pack" finished with status code 200.
verbose 48.378 Performing "GET" request to "https://raw.githubusercontent.com/morrisjs/morris.js/d5cf1410eda7055eaf2c7d218d4cb24ec5ed55c8/package.json".
verbose 48.634 Request "https://raw.githubusercontent.com/morrisjs/morris.js/d5cf1410eda7055eaf2c7d218d4cb24ec5ed55c8/package.json" finished with status code 200.
verbose 48.637 Performing "HEAD" request to "https://github.com/DmitryBaranovskiy/raphael".
verbose 49.354 Request "https://github.com/DmitryBaranovskiy/raphael" finished with status code 200.
verbose 49.357 Performing "GET" request to "https://github.com/DmitryBaranovskiy/raphael.git/info/refs?service=git-upload-pack".
verbose 49.911 Request "https://github.com/DmitryBaranovskiy/raphael.git/info/refs?service=git-upload-pack" finished with status code 200.
verbose 49.915 Performing "GET" request to "https://raw.githubusercontent.com/DmitryBaranovskiy/raphael/a29d980d1f6b0aed9a0f18d751a76e862f5fbc4f/package.json".
verbose 50.139 Request "https://raw.githubusercontent.com/DmitryBaranovskiy/raphael/a29d980d1f6b0aed9a0f18d751a76e862f5fbc4f/package.json" finished with status code 200.
verbose 52.398 Error: Command failed.
Exit code: 128
Command: git
Arguments: ls-remote --tags --heads git://github.com/adobe-webplatform/eve.git
Directory: C:\<path-here>\defectdojo\components
Output:
fatal: unable to connect to github.com:
github.com[0: 192.30.253.112]: errno=No such file or directory
github.com[1: 192.30.253.113]: errno=No such file or directory
    at ProcessTermError.Error (native)
    at ProcessTermError.MessageError (C:\Program Files (x86)\Yarn\lib\cli.js:186:110)
    at new ProcessTermError (C:\Program Files (x86)\Yarn\lib\cli.js:226:113)
    at ChildProcess.<anonymous> (C:\Program Files (x86)\Yarn\lib\cli.js:30281:17)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:191:7)
    at maybeClose (internal/child_process.js:891:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
error An unexpected error occurred: "Command failed.
Exit code: 128
Command: git
Arguments: ls-remote --tags --heads git://github.com/adobe-webplatform/eve.git
Directory: C:\\<path-here>\\defectdojo\\components
Output:
fatal: unable to connect to github.com:
github.com[0: 192.30.253.112]: errno=No such file or directory
github.com[1: 192.30.253.113]: errno=No such file or directory".
info If you think this is a bug, please open a bug report with the information provided in "C:\\<path-here>\\defectdojo\\components\\yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

$
```

The failing dependency gets removed by https://github.com/DmitryBaranovskiy/raphael/commit/be2660dceda7845fda7543871c3c58db31828b85#diff-b9cfc7f2cdf78a7f4b91a753d10865a2 included in version 2.2.2 of raphael.

So, it seems that updating the raphael dependency at [package.json:31](https://github.com/DefectDojo/django-DefectDojo/blob/a2aefd4e5d8798ef8a7a49cd0e487f788c8c035d/components/package.json#L31) resolves the issue. 
However, I don't know whether this raphael version is still compatible with everything else...

**Comment by maennel** [comment]
With my limited frontend knowledge, I think we can even get rid of the Raphael-dependency, since it's [included in "justgage"](https://github.com/toorshia/justgage/blob/master/package.json) and included into DefectDojo templates [only through "justgage"](https://github.com/DefectDojo/django-DefectDojo/search?q=raphael&type=Code&utf8=%E2%9C%93).

Could anyone with more insight into the JS-framework jungle please validate this?
